src/Middleware/HttpLogging/src/PublicAPI.Unshipped.txt

```diff
@@ -37,6 +37,46 @@ Microsoft.AspNetCore.HttpLogging.MediaTypeOptions.AddBinary(string! contentType)
 Microsoft.AspNetCore.HttpLogging.MediaTypeOptions.AddText(string! contentType) -> void
 Microsoft.AspNetCore.HttpLogging.MediaTypeOptions.AddText(string! contentType, System.Text.Encoding! encoding) -> void
 Microsoft.AspNetCore.HttpLogging.MediaTypeOptions.Clear() -> void
+Microsoft.AspNetCore.HttpLogging.W3CLoggerOptions
+Microsoft.AspNetCore.HttpLogging.W3CLoggerOptions.FileName.get -> string!
+Microsoft.AspNetCore.HttpLogging.W3CLoggerOptions.FileName.set -> void
+Microsoft.AspNetCore.HttpLogging.W3CLoggerOptions.FileSizeLimit.get -> int?
+Microsoft.AspNetCore.HttpLogging.W3CLoggerOptions.FileSizeLimit.set -> void
+Microsoft.AspNetCore.HttpLogging.W3CLoggerOptions.FlushPeriod.get -> System.TimeSpan
+Microsoft.AspNetCore.HttpLogging.W3CLoggerOptions.FlushPeriod.set -> void
+Microsoft.AspNetCore.HttpLogging.W3CLoggerOptions.LogDirectory.get -> string!
+Microsoft.AspNetCore.HttpLogging.W3CLoggerOptions.LogDirectory.set -> void
+Microsoft.AspNetCore.HttpLogging.W3CLoggerOptions.LoggingFields.get -> Microsoft.AspNetCore.HttpLogging.W3CLoggingFields
+Microsoft.AspNetCore.HttpLogging.W3CLoggerOptions.LoggingFields.set -> void
+Microsoft.AspNetCore.HttpLogging.W3CLoggerOptions.RetainedFileCountLimit.get -> int?
+Microsoft.AspNetCore.HttpLogging.W3CLoggerOptions.RetainedFileCountLimit.set -> void
+Microsoft.AspNetCore.HttpLogging.W3CLoggerOptions.W3CLoggerOptions() -> void
+Microsoft.AspNetCore.HttpLogging.W3CLoggingFields
+Microsoft.AspNetCore.HttpLogging.W3CLoggingFields.All = Microsoft.AspNetCore.HttpLogging.W3CLoggingFields.Date | Microsoft.AspNetCore.HttpLogging.W3CLoggingFields.Time | Microsoft.AspNetCore.HttpLogging.W3CLoggingFields.UserName | Microsoft.AspNetCore.HttpLogging.W3CLoggingFields.ResponseHeaders | Microsoft.AspNetCore.HttpLogging.W3CLoggingFields.ConnectionInfoFields | Microsoft.AspNetCore.HttpLogging.W3CLoggingFields.ProtocolStatus | Microsoft.AspNetCore.HttpLogging.W3CLoggingFields.TimeTaken | Microsoft.AspNetCore.HttpLogging.W3CLoggingFields.Request -> Microsoft.AspNetCore.HttpLogging.W3CLoggingFields
+Microsoft.AspNetCore.HttpLogging.W3CLoggingFields.ClientIpAddress = 4 -> Microsoft.AspNetCore.HttpLogging.W3CLoggingFields
+Microsoft.AspNetCore.HttpLogging.W3CLoggingFields.ConnectionInfoFields = Microsoft.AspNetCore.HttpLogging.W3CLoggingFields.ClientIpAddress | Microsoft.AspNetCore.HttpLogging.W3CLoggingFields.ServerIpAddress | Microsoft.AspNetCore.HttpLogging.W3CLoggingFields.ServerPort -> Microsoft.AspNetCore.HttpLogging.W3CLoggingFields
+Microsoft.AspNetCore.HttpLogging.W3CLoggingFields.Cookie = 32768 -> Microsoft.AspNetCore.HttpLogging.W3CLoggingFields
```

Comment:
Is this just the name or the contents as well?

Reply:
Name & contents, as per the spec: https://www.loganalyzer.net/log-analyzer/w3c-extended.html